BookingsAPI/Bookings.API/Controllers/SuitabilityAnswersController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Bookings.Api.Contract.Responses;
+using Bookings.API.Mappings;
+using Bookings.DAL.Queries;
+using Bookings.DAL.Queries.Core;
+using Bookings.Domain.Participants;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Bookings.API.Controllers
+{
+    [Produces("application/json")]
+    [Route("suitability-answers")]
+    [ApiController]
+    public class SuitabilityAnswersController : Controller
+    {
+        private const string DefaultCursor = "0";
+        private const int DefaultLimit = 100;
+
+        private readonly IQueryHandler _queryHandler;
+
+        public SuitabilityAnswersController(IQueryHandler queryHandler)
+        {
+            _queryHandler = queryHandler;
+        }
+
+        /// <summary>
+        ///     Get a cursor based list of suitability answers
+        /// </summary>
+        /// <param name="cursor">Cursor specifying from which entries to read next page, is defaulted if not specified</param>
+        /// <param name="limit">The max number hearings records to return.</param>
+        /// <returns>The list of latest suitability answers for participants</returns>
+        [HttpGet]
+        [SwaggerOperation(OperationId = "GetSuitabilityAnswers")]
+        [ProducesResponseType(typeof(SuitabilityAnswersResponse), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
```

Comment:
Since this will never return `NotFound` I think you can remove that.

Reply:
Yes, good point